core/src/main/java/nl/nn/adapterframework/stream/Message.java

```diff
@@ -414,4 +414,37 @@ private void readObject(ObjectInputStream stream) throws IOException, ClassNotFo
 		stream.defaultReadObject();
 	}
 
+	/**
+	 * @return Message size or -1 if it can't determine the size.
+	 */
+	public long size() {
+		if(request == null) {
+			return -1;
```

Comment:
Is dit dan niet 0? -1 betekent "ik weet het niet"

Reply:
Ja potverdorie je hebt helemaal gelijk